app/models/service_template_ansible_playbook.rb

```diff
@@ -74,15 +72,15 @@ def self.create_job_templates(service_name, description, config_info, auth_user)
   def self.create_job_template(name, description, info, auth_user)
     tower, params = build_parameter_list(name, description, info)
 
-    task_id = ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript.create_in_provider_queue(tower.id, params, auth_user)
+    task_id = job_template_class.create_in_provider_queue(tower.id, params, auth_user)
```

Comment:
I think before the change the readability is better although the line looks longer.